src/sideEffect/saga/crudFetch.js

```diff
@@ -34,11 +34,11 @@ const crudFetch = (restClient, successSideEffects = () => [], failureSideEffects
             }
         }
         const sideEffects = successSideEffects(type, meta.resource, payload, response);
-        yield put({ type: FETCH_END });
         yield [
             ...sideEffects.map(a => put(a)),
             put({ type: `${type}_SUCCESS`, payload: response, meta }),
         ];
+        yield put({ type: FETCH_END });
```

Comment:
this helps preventing useless renders in the list by simply watching the `isLoading` prop